### PR TITLE
return zeros for unmapped lbas

### PR DESCRIPTION
**Pull request by raakella1**

### CODE REVIEW

File: src/lib/volume/volume.cpp [hunk 1]
```diff
@@ -417,6 +423,17 @@ void Volume::submit_read_to_backend(read_blks_list_t const& blks_to_read, const
         prev_lba = start_lba;
         prev_nblks = blkids.blk_count();
     }
+    // if there are any holes at the end, fill them with zeros
+    if (prev_lba + prev_nblks < req->end_lba() + 1) {
```
Comment (sanebay) [inline src/lib/volume/volume.cpp:427]: Do we need a +1 here ?

Comment (raakella1) [reply src/lib/volume/volume.cpp:427]: If prev_lba = 3 and nblks = 2 and end lba is 8, here is how it looks:
lba # 3, 4 (previous data); lba# 5, 6, 7, 8 zeros. There will be 4 zeros not 3

**Review by sanebay — APPROVED** [review]
DIscussed with Arjun. Tests disabled will be enabled after HS fix is merged. More UT's to handle corner cases during read in next PR.
